.github/workflows/nightly-release.yaml

```diff
@@ -31,19 +31,19 @@ jobs:
       - name: Set dev version
         id: version
         run: |
-            versionFile="pinecone/__version__"
-            currentDate=$(date +%Y%m%d%H%M%S)
-            versionNumber=$(cat $versionFile)
-            devVersion="${versionNumber}.dev${currentDate}"
-            echo "$devVersion" > $versionFile
+          versionFile="pinecone/__version__"
+          currentDate=$(date +%Y%m%d%H%M%S)
+          versionNumber=$(cat $versionFile)
+          devVersion="${versionNumber}.dev${currentDate}"
+          echo "$devVersion" > $versionFile
```

Comment:
This all looks reasonable to me. I was trying to think of some edge cases where it would be possible to perform a partial or incomplete version bump, but theoretically if any of these subcommand were to fail, the step itself would fail and then the GitHub Action job would fail.